forge-game/src/main/java/forge/game/combat/Combat.java

```diff
@@ -72,7 +72,7 @@ public Combat(final Player attacker) {
         initConstraints();
     }
 
-    public Combat(Combat combat, GameObjectMap map) {
+    public Combat(Combat combat, IEntityMap map) {
```

Comment:
Need to do some more testing about rolling back combat. Just had some weirdness occur where nothing died when canceling a spell during combat.

Reply:
Ok, the map was grabbing the wrong game when restoring. This should be fixed now.

Reply:
Can't confirm
To me it looks like canceling (say _Doubling Cube_) any ability during combat prevents damage from every creature involved...